project/ChatBuild.scala

```diff
@@ -44,8 +45,20 @@ object ChatBuild extends Build {
             .map((f1, f2) => Seq(f1.data, f2.data)),
         watchSources <++= (watchSources in frontend)
       )
+      .dependsOn(sharedJvm)
+
+  lazy val shared = (crossProject.crossType(CrossType.Pure) in file ("shared")).
+    settings(
+      scalaVersion:=scalaV,
+      libraryDependencies +="org.scala-js" %% "scalajs-stubs" % scalaJSVersion % "provided"
```

Comment:
It's boilerplate I've seen in [other](https://github.com/vmunier/play-with-scalajs-example/blob/master/build.sbt) [projects](https://github.com/ochrons/scalajs-spa-tutorial/blob/master/build.sbt).

Though, looking at the [docs](http://www.scala-js.org/doc/sbt/cross-building.html) myself, it seems there may be simpler ways of doing it.

Let me know if you'd like to change it.